### ISSUE TITLE
Website from the README (http://visi.kenshoto.com/) is down

### ISSUE THREAD

**Issue by pfalcon** [vivisect/vivisect#203]
README refers to http://visi.kenshoto.com/ for additional information, but that site can't be contacted. More specifically, "visi.kenshoto.com" or "kenshoto.com" domain names can't be resolved, including when using Google's DNS 8.8.8.8. This happens at least for 2nd day.

Note that without that site, the README barely gives any information about the project, for example Vdb's descriptions starts with:

> As in previous vdb releases, the command python vdbbin

How is that relevant for a new user?

And Vivisect is described as:

> Fairly un-documented static analysis / emulation / symbolik analysis framework

**Comment by follower** [comment]
FWIW archive.org has snapshots of the page to which that URL originally redirected:

 * Most recent: [http://visi.kenshoto.com/viki/MainPage (on archive.org)](http://web.archive.org/web/20161225225334/http://visi.kenshoto.com:80/viki/MainPage)

Snapshots of sub pages also seem to be present.

**Comment by 9527KGG** [comment]
Is it possible to move the doc to github? just wondering~

**Comment by rakuy0** [comment]
Possibly? Though it be be more worth it to create new ones since I don't have ready access to them, so I have no idea what state they're in.

They're also definitely pre-python3 vivisect, so even if we migrated them, they'd need some touch ups.

But honestly, me or @atlas0fd00m need to carve out time to just make new docs. Enough has changed in the last few years. We've just been super busy. It's been a long time TODO, but I'm hoping now that python3 is off my plate and some of atlas's big PRs are coming down the home stretch we can carve out time for them.

TL;DR: Been super busy, gonna circle back once some big ticket items are done to write some fresher examples.

(and sorry about the delay in response. I completely missed the notifications about this thread)
